pkgs/development/libraries/libgsf/default.nix

```diff
@@ -31,6 +32,14 @@ stdenv.mkDerivation rec {
     hash = "sha256-uSi2/pZiST07YutU8SHNoY2LifEQhohQeyaH9spyG2s=";
   };
 
+  patches = [
+    # Conditionally uses nanohttp if available. Fixes build with libxml2 2.13
+    (fetchpatch2 {
```

Comment:
This will break when Git backend decides to bump hash length https://github.com/NixOS/nixpkgs/issues/257446

Reply:
GitLab doesn’t support `?full_index=1`, but using `.diff` instead of `.patch` appears to have the same effect. I’ll switch it to `.diff`.